sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/recognizeEntitiesForBatchListCountryHint.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
Should these recordings have valid data recorded, rather than be deleted?

Reply:
we don't have this test